src/auth/session/storage/mysql.ts

```diff
@@ -121,9 +121,12 @@ export class MySQLSessionStorage implements SessionStorage {
 
   private async hasSessionTable(): Promise<boolean> {
     const query = `
-      SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = ?;
+      SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = ? AND TABLE_SCHEMA = ?;
```

Comment:
Since we only care about the number of rows that this request returns, I would suggest to use a column for the query, eg. `TABLE_NAME` so that we are more likely to get the value from an index'ed column and therefore make the request a bit more efficient.

So the request would become `SELECT TABLE_NAME FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = ? AND TABLE_SCHEMA = ?;`

Reply:
Good idea!